Godeps/Godeps.json

```diff
@@ -406,6 +406,11 @@
 			"Rev": "a1b8ba5163b7f041b22761461eabd02b70d1f824"
 		},
 		{
+			"ImportPath": "github.com/gogo/protobuf/proto",
+			"Comment": "v0.1-125-g82d16f7",
```

Comment:
I used the exact same version as appc/spec https://github.com/appc/spec/blob/v0.8.3/Godeps/Godeps.json#L15